tests-web/www/js/main.js

```diff
@@ -55,16 +55,18 @@ export class StreetExplorer {
 
     // Wire up the import button
     const importButton = document.getElementById("import-view");
-    importButton.onclick = async () => {
-      if (app.map.getZoom() < 15) {
-        window.alert("Zoom in more to import");
-        return;
-      }
-
-      await app.setCurrentTest((app) =>
-        TestCase.importCurrentView(app, importButton)
-      );
-    };
+    if (importButton) {
```

Comment:
Could it ever be missing? I need to read up about initialization order on the web. This code is triggered in a `<script type="module">` in `<head>`. The button is defined in `<body>`. Do we need to specify something to make sure the body is created first?

Reply:
When loading the page with a test in the URL, the code just above awaits the loading of the test, which replaces the button.